src/deploy/functions/release/fabricator.ts

```diff
@@ -233,8 +233,16 @@ export class Fabricator {
           })
           .catch(rethrowAs(endpoint, "set invoker"));
       }
+    } else if (backend.isCallableTriggered(endpoint)) {
+      // Callable functions should always be public
+      const invoker = ["public"];
```

Comment:
nit* Personal preference, but I'd inline this value instead of creating an intermediate var.

Same comment for L310

Reply:
Sure - makes sense.